Sources/SnapshotTesting/Snapshotting/SwiftUIView.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  File.swift
+//  
+//
+//  Created by Nathan Mann on 8/17/19.
+//
+
+import Foundation
+import SwiftUI
+
+@available(iOS 13.0, *)
```

Comment:
Let's scope these things to all SwiftUI platforms.

```suggestion
@available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
```

Might even need a `macCatalyst 13.0` in there but not sure if that works.

Reply:
(I've omitted watchOS because it's my understanding that you can't XCTest it. If that's changed, though, would love to add support!)